assets/src/application/ui/layout/entityList/filterBar/style.scss

```diff
@@ -13,8 +13,76 @@
 	box-sizing: border-box;
 	color: var(--ee-default-text-color);
 	margin: 0 0 var(--ee-margin-small);
-	overflow: hidden;
```

Comment:
this is the main change in this file;
the rest was just css -> scss transformation